REQUIREMENTS.txt

```diff
@@ -13,3 +13,8 @@ pydantic
 uvicorn
 jinja2
 imgkit
+databases
+
+asyncssh
+socket
+asyncio
```

Comment:
no need to have socket or asyncio in the requirements, they're part of the std lib

Reply:
fixed